lib/elixir/lib/version.ex

```diff
@@ -51,10 +51,12 @@ defmodule Version do
 
       "~> 2.0.0"
 
-  `~>` will never include pre-release versions of its upper bound.
+  `~>` will never include pre-release versions of its upper bound,
+  regardless of the usage of the `allow_pre:` option, or whether the operand
```

Comment:
Maybe "of the `:allow_pre` option"?

Reply:
Sweet. I will submit a PR fixing all similar issues. Thanks for pointing it out @fertapric